crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -74,9 +72,7 @@ def upgrade_ha_repo?
     end
 
     def upgrade_ceph_repo_needed?
-      return true if Proposal.where(barclamp: "ceph").first
-
-      false
+      !Proposal.where(barclamp: "ceph").empty?
```

Comment:
Yes, it does, but the end result is the same. It "returns true if there are any records.".